pkg/grpc/grpc.go

```diff
@@ -14,7 +14,7 @@
    limitations under the License.
 */
 
-package errdefs
+package grpc
```

Comment:
errdefs.FromGRPC should be now called like grpc.ToNative ?

Reply:
Yeah, `grpc.FromGRPC` can be redundant. `grpc.ToNative` is nice naming imo. Pushed to try it out.

Reply:
Nit: maybe `grpcerror` or something?
I just dislike having package name conflicts that makes goimports trip up.